src/css/main.less

```diff
@@ -12,6 +12,15 @@
        url('fonts/SourceSansPro-Regular.otf.woff') format('woff');
 }
 
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: normal;
+  font-weight: bold;
+  src: local('Source Sans Pro Bold'),
+  local('SourceSansPro-Bold'),
+  url('fonts/SourceSansPro-Bold.otf.woff') format('woff');
```

Comment:
We should update the Gulpfile to copy the bold font as well as the regular, or it won't be available here. See [here](https://github.com/mesosphere/marathon-ui/commit/1e4411ebefb41a71a12974c3a3ca4b51085b10d6#diff-b9e12334e9eafd8341a6107dd98510c9R146) for an example.